src/debate/debate.js

```diff
@@ -163,6 +185,18 @@ export class Debate {
             return;
         }
 
+        if (question.isOpenQuestion) {
+            logger.debug(`Question with id (${questionId}) is not an open question.`);
+            callback(false);
+            return;
+        }
```

Comment:
I assume that `question.isOpenQuestion` is set to `true` if the current question is an oppened one. Then why is the logger debuging 'Question with id (${questionId}) is **not** an open question.' if `question.isOpenQuestion` equal `true` ? \
Am I missing something ?

Reply:
It was a copy paste mistake. I fixed it in the latest commit.

``logger.debug(`Question with id (${questionId}) is an open question and not a closed question.`);``